File: apps/problems/Peak-element/boilerPlate-Full/function.js

  ##USER_CODE_HERE##
  
  const input = require('fs').readFileSync('/dev/stdin', 'utf8').trim().split('\n').join(' ').split(' ');
  const arr = parseInt(input.shift());
  const result = peekElement(arr);
  console.log(result);
      

File: apps/problems/Peak-element/boilerPlate-Full/function.rs

  use std::io::{self, BufRead};
  
  ##USER_CODE_HERE##
  
  fn main() {
    let stdin = io::stdin();
    let mut input = stdin.lock().lines().map(|l| l.unwrap());
    let arr: i32 = input.next().unwrap().parse().unwrap();
    let result = peekElement(arr);
    println!("{}", result);
  }
      